addon/mixins/sortable-item.js

```diff
@@ -112,32 +123,71 @@ export default Mixin.create({
     @property y
     @type Number
   */
-  y: computed(function(_, value) {
-    if (arguments.length === 2 && value !== this._y) {
-      this._y = value;
-      this._scheduleApplyPosition();
-    }
-
-    if (this._y === undefined) {
-      this._y = this.element.offsetTop;
-    }
+  y: computed({
+    get: function() {
+      if (this._y === undefined) {
+        this._y = this.element.offsetTop;
+      }
 
-    return this._y;
+      return this._y;
+    },
+    set: function(_, value) {
+      if (value !== this._y) {
+        this._y = value;
+        this._scheduleApplyPosition();
+      }
+    },
   }).volatile(),
 
   /**
+    Horizontal position of the item.
+
+    @property x
+    @type Number
+  */
+  x: computed({
+    get: function() {
+      if (this._x === undefined) {
+        this._x = this.element.offsetLeft;
+      }
+
+      return this._x;
+    },
+    set: function(_, value) {
+      if (value !== this._x) {
+        this._x = value;
+        this._scheduleApplyPosition();
+      }
+    },
+  }).volatile(),
```

Comment:
See general comment about new-style computed.

And could we sneak a newline here between the method and follow docstring?